lib/active_merchant/billing/gateways/blue_snap.rb

```diff
@@ -564,7 +564,7 @@ def bad_authentication_response
         { 'description' => 'Unable to authenticate.  Please check your credentials.' }
       end
 
-      def forbidden_response(body)
+      def generic_error_response(body)
```

Comment:
not sure about this name - I was going to add another method specifically for rate limiting errors, but I wanted to return the same thing. Figured I'd use this method for both types